gapic-generator-ads/Rakefile

```diff
@@ -143,6 +143,18 @@ def generate_ads_templates service
   samples = samples_for service
   protoc_cmd << "--ruby_gapic_opt='samples=#{samples.join ';'}'" if samples
 
+  # Add optional grpc service config
+  grpc_service_config = grpc_service_config_for service
+  unless grpc_service_config.empty?
+    protoc_cmd << "--ruby_gapic_opt='grpc_service_config=#{grpc_service_config.join ';'}'"
```

Comment:
It's really unclear to me when these option names have underscores (as here) vs hyphens (as below for service-yaml). Can we consider one or the other canonical?